src/Notepads/NotepadsMainPage.StatusBar.cs

```diff
@@ -195,6 +208,63 @@ private async void ReloadFileFromDisk(object sender, RoutedEventArgs e)
             }
         }
 
+        /*private void LineColumnIndicatoFlyoutSelection_OnClick(object sender, RoutedEventArgs e)
+        {
+            if (sender is CheckBox checkBox)
+            {
+                switch ((string)checkBox.Name)
+                {
+                    case "WrapWord":
+                        EditorSettingsService.EditorDefaultTextWrapping = (bool)WrapWord.IsChecked ? TextWrapping.Wrap : TextWrapping.NoWrap;
+                        EditorSettingsService.IsLineHighlighterEnabled = (bool)HighlightCurrentLine.IsChecked;
+                        break;
+                    case "HighlightCurrentLine":
+                        EditorSettingsService.IsLineHighlighterEnabled = (bool)HighlightCurrentLine.IsChecked;
+                        break;
+                }
+            }
+            else if (sender is AppBarButton appBarButton && appBarButton.Name == "GoToLine")
+            {
+                LineColumnIndicatorFlyout.Hide();
+                NotepadsCore.GetSelectedTextEditor()?.ShowGoToControl();
+            }
+            else
+                return;
+        }*/
+
+        private void FontZoomIndicatoFlyoutSelection_OnClick(object sender, RoutedEventArgs e)
+        {
+            if (!(sender is AppBarButton button)) return;
+
+            var selectedTextEditor = NotepadsCore.GetSelectedTextEditor();
+            if (selectedTextEditor == null) return;
+
+            switch ((string)button.Name)
+            {
+                case "ZoomIn":
+                    selectedTextEditor.SetCurrentFontZoomFactor(FontZoomSlider.Value + 10);
+                    break;
+                case "ZoomOut":
+                    selectedTextEditor.SetCurrentFontZoomFactor(FontZoomSlider.Value - 10);
+                    break;
+                case "RestoreDefaultZoom":
+                    selectedTextEditor.SetCurrentFontZoomFactor(100);
+                    FontZoomIndicatorFlyout.Hide();
+                    break;
+            }
+        }
+
+        private void FontZoomSlider_ValueChanged(object sender, RangeBaseValueChangedEventArgs e)
+        {
+            if (!(sender is Slider item)) return;
+
+            var selectedTextEditor = NotepadsCore.GetSelectedTextEditor();
+            if (selectedTextEditor == null) return;
+
+            if (e.NewValue != e.OldValue)
```

Comment:
This will never be false since range value is a double.
You can round them into integer and compare or write something like this:
if (Math.Abs(e.NewValue - e.OldValue) > TOLERANCE)